server/src/main/java/com/linecorp/centraldogma/server/internal/admin/service/RepositoryService.java

```diff
@@ -171,14 +170,14 @@ public HttpResponse deleteFile(@Param("projectName") String projectName,
      */
     @Get("regex:/projects/(?<projectName>[^/]+)/repositories/(?<repoName>[^/]+)" +
          "/history(?<path>/.*$)")
-    public CompletionStage<List<CommitDto>> getHistory(@Param("projectName") String projectName,
-                                                       @Param("repoName") String repoName,
-                                                       @Param("path") String path,
-                                                       @Param("from") Optional<String> from,
-                                                       @Param("to") Optional<String> to) {
+    public CompletionStage<List<CommitDto>> getHistory(@Param String projectName,
+                                                       @Param String repoName,
+                                                       @Param String path,
+                                                       @Param @Default("-1") String from,
+                                                       @Param @Default("-1") String to) {
```

Comment:
The original fallback was `1`.
```suggestion
                                                       @Param @Default("1") String to) {
```

Reply:
Oh no! Nice catch :+1: